### ISSUE TITLE
Unable to retrieve nonce in the token verification

### ISSUE THREAD

**Issue by xymontana** [bytesbay/web3-token#49]
Hi, I've put in the nonce and request_id during the Web3Token.sign, but the nonce and the request_id was not returned during Web3Token.verify. Any ideas on how to solve this?

**Comment by bytesbay** [comment]
@xymontana Hey let me check

**Comment by bytesbay** [comment]
Fixed in [57a09e1](https://github.com/bytesbay/web3-token/commit/57a09e10a0ae686fbd2e1a2fa73de38c941ce1da). Thanks alot for the report. Just update the package to 1.0.4.

**Comment by xymontana** [comment]
thank you!

**Comment by xymontana** [comment]
> Fixed in [57a09e1](https://github.com/bytesbay/web3-token/commit/57a09e10a0ae686fbd2e1a2fa73de38c941ce1da). Thanks alot for the report. Just update the package to 1.0.4.

Hi I see that you've fixed the request-id. But regarding the nonce, shouldn't it be like this:
```
   if (params.nonce) {
        body.nonce = params.nonce;
    }
```
